src/main/java/com/google/sps/servlets/CSVServlet.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps.servlets;
+ 
+import com.google.api.client.auth.oauth2.TokenResponseException;
+import com.google.appengine.api.datastore.PreparedQuery.TooManyResultsException;
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.data.ChromeOSDevice;
+import com.google.sps.gson.Json;
+import com.google.sps.servlets.Util;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.StringBuilder;
+import java.security.GeneralSecurityException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+ 
+@WebServlet("/csv")
+public class CSVServlet extends HttpServlet {
+ 
+  private UserService userService = UserServiceFactory.getUserService();
+  private Util utilObj = new Util();
+  public final String LOGIN_URL = "/login";
```

Comment:
These constants seem to appear in multiple files. Consider putting them in a shared place.